PontoFacil/PontoFacil/Views/PlanningPage.xaml.cs

```diff
@@ -22,9 +12,25 @@ namespace PontoFacil.Views
     /// </summary>
     public sealed partial class PlanningPage : Page
     {
+        private DateTime accumulatedHoursValue = DateTime.MinValue;
```

Comment:
I think it is better to initialize the attributes in the constructor, only const or readonly attributes should be initialized like this.